zenml/cli/config.py

```diff
@@ -54,6 +55,18 @@ def opt_out(config):
     click.echo('Opted out of analytics.')
 
 
+@config.command("list")
+def list_config():
+    """Print the current ZenML config to the command line"""
+    try:
+        repo: Repository = Repository.get_instance()
```

Comment:
As this pattern repeats so much, you could consider refactoring into a seperate utils funciton. However, I do not think the try catch is neccessary, as the `get_instance()` should be throwing the exception anyway.

If the intention is to print a nicer intention then still please refactor to a seperate function

Reply:
Fixed in commit 053bd51.